static/files/c2c/taxii-1/example.tf

```diff
@@ -0,0 +1,20 @@
+resource "sumologic_cloud_to_cloud_source" "taxii2_source" {
+  collector_id = sumologic_collector.collector.id
+  schema_ref = {
+    type = "TAXII 2 Client"
```

Comment:
@carlos-castillo-jask, do you need to change this line in the example and download from "TAXII 2 Client" to "TAXII 1 Client"?

Reply:
I also just noticed "taxii2_source" in the example and download. You'll want to fix that too.